Rubberduck.Resources/RubberduckUI.de.resx

```diff
@@ -1686,15 +1686,15 @@ Import abgebrochen.</value>
     <value>Es ist in VBA nicht möglich nur die Groß- und Kleinschreibung eines Namens zu ändern.</value>
   </data>
   <data name="MoveFolders_SameNameSubfolder" xml:space="preserve">
-    <value>Der Zielordner '{0}' enthält bereits einen Unterordner '{1}'. Die Ordner werden verschmolzen, wenn der Ordner verschoben wird.
+    <value>Der Zielordner '{0}' enthält bereits einen Unterordner '{1}'. Die Ordner werden zusammengelegt, wenn der Ordner verschoben wird.
```

Comment:
Ich halte _verschmolzen_ hier nicht wirklich für seltsam, wie von dir angelerkt, aber die Ersetzung ist auch ok für mich.

Reply:
Beim Pull Request kam mir dann die Windows Formulierung für Ordner in den Sinn. Da heißt es ja "wollen Sie Ordner A in Odner B integrieren,wäre dann einheitlich.